packages/cli/compiler/win32.ts

```diff
@@ -1,14 +1,60 @@
 import { execSync } from 'node:child_process';
```

Comment:
why didnt you use $ from zx?
import { $ } from 'zx';

Reply:
i didn't want to add more dependencies.
used only native operations for now.
we can improve it later.

Reply:
it is already installed on the global monorepo as dev dep